README.md

```diff
@@ -133,10 +133,232 @@ Tests may be run individually (use `--help` for usage), or collectively via the
 ## Tests
 
 ### heap
+
+In x86 architecture, the stack grows up and towards the heap (which by
+convention grows in the opposite direction to the stack).
+![Heap and stack](heap-stack.png)
```

Comment:
I don't know how the rendering will turn out, but if you add a blank line before the heap/stack picture, will it bump it down a little bit (to even out the vertical space above and below it).